Tests/Clappr_Tests/AVFoundationPlaybackTests.swift

```diff
@@ -933,6 +933,207 @@ class AVFoundationPlaybackTests: QuickSpec {
+
+                            expect(playback.isPaused).to(beTrue())
+                            expect(playback.currentState).to(equal(.paused))
+                        }
+                    }
+
+                    context("when stop is called") {
+                        it("changes state to idle") {
+                            let playback = AVFoundationPlayback()
+
+                            playback.play()
+                            playback.pause()
+                            playback.stop()
+
+                            expect(playback.currentState).to(equal(.idle))
+                        }
+                    }
+
+                    context("when is not likely to keep up") {
+                        it("changes state to buffering") {
+                            let playback = AVFoundationPlayback()
+                            playback.player = AVPlayerStub()
+
+                            playback.play()
+                            playback.pause()
+                            playback.play()
+
+                            expect(playback.isBuffering).to(beTrue())
+                            expect(playback.currentState).to(equal(.buffering))
+                        }
+                    }
+                }
+
+                describe("#stalled") {
+                    context("when seek is called") {
+                        it("keeps buffering state") {
+                            let playback = AVFoundationPlayback()
+                            playback.player = AVPlayerStub()
+
+                            playback.play()
+                            playback.seek(10)
+                            
+                            expect(playback.isBuffering).to(beTrue())
+                            expect(playback.currentState).to(equal(.buffering))
+                        }
+                    }
+                    
+                    context("when paused is called") {
+                        it("changes state to paused") {
+                            let playback = AVFoundationPlayback()
+                            
+                            playback.play()
+                            playback.pause()
+                            
+                            expect(playback.isPaused).to(beTrue())
+                            expect(playback.currentState).to(equal(.paused))
+                        }
+                    }
+                    
+                    context("when stop is called") {
+                        it("changes state to idle") {
+                            let playback = AVFoundationPlayback()
+
+                            playback.play()
+                            playback.stop()
+                            
+                            expect(playback.currentState).to(equal(.idle))
+                        }
+                    }
+                    
+                    context("when is likely to keep up") {
+                        it("has isPlaying as true") {
```

Comment:
What is the difference between this test and the second `it`? It seems that is the same.

Reply:
You are right, we've just removed it.